src/components/Categories.js

```diff
@@ -1,10 +1,17 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import { checkStatus } from '../redux/categories/categories';
-import store from '../redux/configureStore';
 
 const Categories = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(checkStatus());
+  }, []);
+  const categories = useSelector((state) => state.categories);
+
   const clickCheckStatus = () => {
-    store.dispatch(checkStatus());
-    console.log(store.getState().categories);
+    dispatch(checkStatus());
+    console.log(categories);
```

Comment:
- [ ] For best practice, kindly remove all `conslole.log()` it makes your code slower

Reply:
I don't like this review. console.log is for the coming-up sections of the projects. But I'll remove it temporarily to get approved.

Reply:
@HtetNaing0814, you should not commit `console.log()` to your Github repo,  there is no reason to use `console.log()` for anything in this project other than debugging